.github/workflows/build-docker.yml

```diff
@@ -4,11 +4,17 @@ on:
   push:
     branches:
       - main
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
 
 jobs:
   build-docker:
     name: Build D-Voting Docker images
     runs-on: ubuntu-22.04
+    if: '! github.event.pull_request.draft'
```

Comment:
I don't agree with that - I can very clearly see wanting to create the Docker image for a PR that's not yet ready for testing purposes

it's not merged anyway, I don't see the advantage of creating a third case - either it's on main, then it should have the latest tag or it is not, then it gets a "non-latest" tag anyway

Reply:
I thought of the poor github servers and the environment: if you mark a PR as `draft`, then it's mostly to show that you're currently working on this feature/fix/whatever. All tests happen locally on your computer, you create the images locally, etc.

Only once you mark it as ready for review do your fellow engineers need to have access to the docker images.

Reply:
OK - I mainly want to avoid overcomplicating this

Reply:
If we get confused by it, we can still remove it later.